### DISCUSSION TITLE
Stack overflowed

### DISCUSSION THREAD

**Discussion by nocountryforusreal** [Totodore/socketioxide#313] [Q&A] [answered]
After service started for some days, it suddenly shut down by following error. Anyone facing this issue?

> axum tokio listener User thread 'tokio-runtime-worker' has overflowed its stack

**Comment by Totodore** [comment]
> After service started for some days, it suddenly shut down by following error. Anyone facing this issue?
> 
> > axum tokio listener User thread 'tokio-runtime-worker' has overflowed its stack

Could you add more info? A stack trace, the socketioxide version.
Also the kind of traffic you had (how many namespaces/lot of binary data or no/how many event handlers?).
This is not necessarily a socketioxide bug but if it is, these infos might help me on what is the origin of the issue.

Also make sure to not declare too heavy stack allocated data.

**Reply by nocountryforusreal** [reply]
- version 0.12.0
- it's about 200 ns, (register dynamically)
- ~200 ccu 
No stack trace printout. It throw fatal error with above logs then stop application

**Comment by nocountryforusreal** [comment, marked as answer]
it's not socketioxide bug. This come from std::fmt::Display implement on my error type
